src/main/java/com/javapractices/app/Account.java

```diff
@@ -0,0 +1,22 @@
+package com.javapractices.app;
+
+public class Account {
+	double principal, rate;
+	int daysActive, accountType;
+	public static final int STANDARD = 0, BUDGET = 1, PREMIUM = 2,
+			PREMIUM_PLUS = 3;
+
+	public static double calculateFee(Account[] accounts) {
+		double totalFee = 0.0;
+		Account account;
+		for (int i = 0; i < accounts.length; i++) {
+			account = accounts[i];
```

Comment:
It's not wrong. Just wondering. You could use a for each loop instead.

`for(Account account : accounts) {
    // Code here
}`